### ISSUE TITLE
A question about "l1 loss" between pred outcome and all one tensor in trainer

### ISSUE THREAD

**Issue by YoungQuasimodo** [dbash/lasagna#1]
For code in shading_controlnet_trainer.py,line 247`reg_loss = torch.nn.functional.l1_loss(pred_mult_layer, torch.ones_like(pred_mult_layer))
reg_loss += torch.nn.functional.l1_loss(pred_div_layer, torch.ones_like(pred_div_layer))`，
why calculating L1 loss between pred outcome and all one tensor,  is it to minimize over-exposure artifacts? 
Ant it would be greatly appreciated if you could make the dataset and inference code available
